github_connector/views/view_github_team_partner.xml

```diff
@@ -21,13 +21,15 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
             <tree decoration-warning="role == 'member'">
                 <field
                     name="team_id"
-                    invisible="context.get('search_default_team_id')"
```

Comment:
I don't get why this is not working.

Reply:
AFAICS, it is due to the removal of dynamic expressions for the `invisible` and `readonly` attributes in XML for v14. We are forced to use `attrs` instead, and declare it there, and for some reason it is not accepting the syntax you suggested in https://github.com/OCA/interface-github/pull/73#discussion_r668002750

I dig a little and found that this is what we already had to do in https://github.com/OCA/product-attribute/pull/742/commits/78b534eced1166f77525b3de1575df369533b038#diff-68f52c9b5cbb5cad326ecb91bf58f75aa8b5a47d4b82f88b0e1dd3b575ea6df7L11-L20

Reply:
But I still see this expression on v14 code: https://github.com/odoo/odoo/blob/14.0/addons/mail/wizard/invite_view.xml#L17

Reply:
Mmm yes, then is somehting wrong with the migration steps? https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#tasks-to-do-in-the-migration

> In View XML, the `invisible` and `readonly` no longer support dynamic expressions. When using those, use the `attrs` attribute instead

Reply:
I have changed such line to:
```
* In the XML views, the invisible and readonly no longer support dynamic 
  expressions like `state=='draft'`. When using those, use the `attrs` attribute 
  instead. You can only use them for direct context queries like `context.get('invisible')`.
```